kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/util/PipelineDistributedBarrier.java

```diff
@@ -109,12 +116,13 @@ public void removeParentNode(final String parentPath) {
     /**
      * Await until all children node is ready.
      *
-     * @param parentPath parent path
+     * @param jobId job id
      * @param timeout timeout
      * @param timeUnit time unit
      * @return true if the count reached zero and false if the waiting time elapsed before the count reached zero
      */
-    public boolean await(final String parentPath, final long timeout, final TimeUnit timeUnit) {
+    public boolean await(final String jobId, final long timeout, final TimeUnit timeUnit) {
+        String parentPath = PipelineMetaDataNode.getJobBarrierEnablePath(jobId);
```

Comment:
The first parameter is changed from `parentPath` to `jobId`, `parentPath` might be `enable` or `disable` path, and now it's hard-coded `enable` path, is it expected?